services/solr.md

```diff
@@ -1,2 +1,19 @@
+## Data center constraints for our Solr cloud
+Due to latency and throughput issues between our two data centers, all VMs on our Solr cloud must be in a single data center for indexing to run successfully. In an outage, one or more VMs may fail over to the other data center - read operations will continue in this scenario, but reindexing will fail. Once the data center has recovered from the outage, vMotion all VMs in our Solr cloud back to the same data center before reindexing.
```

Comment:
is there a missing word somewhere in `outage, vMotion all VMs in our Solr...`

Reply:
Or is vMotion a verb?

Reply:
Good catch - I have heard 'vMotion' used as a verb, but I think it would be better to avoid jargon here.

Reply:
addressed in 3c6b14ea159875924ef7f3becad3bcd1c0aabe95